include/MBProgressHUD/MBProgressHUD.h

```diff
@@ -0,0 +1 @@
+../../MBProgressHUD.h
```

Comment:
Could you perhaps elaborate on the significance of this file?  Adding some comments seems appropriate.

Reply:
This file is a symbolic link to satisfy the module.modulemap, sorry but I'm not an expert on why, I used that on some others SPM libraries, I don't remember where did I get this.

Reply:
Sorry about not giving to much technical details on this.
I have used this also in other like https://github.com/zxingify/zxingify-objc/pull/489

Reply:
I do remember now where I got this.
I saw this in the real-cocoa repository.
Here is the commit where they add support for SPM https://github.com/realm/realm-cocoa/commit/26e2704278863baf8a0bef0f8cce10ed0ee3d9cd

This is how it works, first we need to create a directory `include`.
Inside the `include` directory, we create a `module.modulemap` and a subdirectory with the library name, in this case `MBProgressHUD`.
The `module.modulemap` will point at the subdirectory with the library name to search for the headers.
Inside subdirectory with the library name, `MBProgressHUD`, we need to create symbolic links to the headers used by the library.
And in the `Package.swift` in the `publicHeadersPath` we point to the `include` directory.
This way the SPM imports correctly the headers.

In the realm commit, here is the [Package.swift](https://github.com/realm/realm-cocoa/blob/963dbe558b802e3b9b78b810dcbceea3b4853ec9/Package.swift), the [include](https://github.com/realm/realm-cocoa/blob/963dbe558b802e3b9b78b810dcbceea3b4853ec9/include) directory with the [module.modulemap](https://github.com/realm/realm-cocoa/blob/963dbe558b802e3b9b78b810dcbceea3b4853ec9/include/module.modulemap) and the project name sub directory, [Realm](https://github.com/realm/realm-cocoa/tree/963dbe558b802e3b9b78b810dcbceea3b4853ec9/include/Realm) in their case, with all the symbolic links to the headers.

Hope it helps to clarify how this works.